src/reference/asciidoc/amqp.adoc

```diff
@@ -4081,6 +4081,38 @@ converter to determine the type.
 IMPORTANT: Starting with version 1.6.11, `Jackson2JsonMessageConverter` and, therefore, `DefaultJackson2JavaTypeMapper` (`DefaultClassMapper`) provide the `trustedPackages` option to overcome https://pivotal.io/security/cve-2017-4995[Serialization Gadgets] vulnerability.
 By default and for backward compatibility, the `Jackson2JsonMessageConverter` trusts all packages -- that is, it uses `*` for the option.
 
+Starting with version 2.4.7, the converter can be configured to return `Optional.empty()` if Jackson returns `null` after deserializing the message body.
+This facilitates `@RabbitListener` s to receive null payloads, in two ways:
+
+====
+[source, java]
+----
+@RabbitListener(queues = "op.1")
+void listen(@Payload(required = false) Thing payload) {
```

Comment:
I wonder if `PayloadMethodArgumentResolver` should treat a `@Nullable` as equal to this one.
Much shorter and intuitive for end-user codding 😄

Reply:
True; feel free to open an issue against Framework :D

Reply:
Here you are: https://github.com/spring-projects/spring-framework/issues/28945 !